admin/assets/csv/multiple_pupils_template.csv

```diff
@@ -0,0 +1 @@
+﻿First name,Middle name(s),Last name,UPN,Date of Birth,Gender
John,Lawrence,Smith,H999200014001,22/05/2005,Male
Maria,Stella,Brown,H999200014002,15/07/2005,Female
```

Comment:
This looks like it is on one line... please confirm.

Reply:
<img width="412" alt="screen shot 2017-09-20 at 1 04 30 pm" src="https://user-images.githubusercontent.com/4175503/30642832-4b40e5b2-9e04-11e7-942e-e30d7daacf47.png">